src/components/ProductTours/AdminOnboardingTours/data/useAdminOnboardingTour.tsx

```diff
@@ -0,0 +1,67 @@
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+import { logError } from '@edx/frontend-platform/logging';
+
+import {
+  ANALYTICS_INSIGHTS_TARGETS,
+  ORGANIZE_LEARNER_TARGETS,
+  TRACK_LEARNER_PROGRESS_TARGETS,
+} from '../constants';
+
+import LmsApiService from '../../../../data/services/LmsApiService';
+import useCreateLearnerProgressFlow from './useCreateLearnerProgressFlow';
+import useCreateAnalyticsFlow from './useCreateAnalyticsFlow';
+import { TourStep } from '../../types';
+import useCreateOrganizeLearnersFlow from './useCreateOrganizeLearnersFlow';
+
+interface UseAdminOnboardingTourProps {
+  adminUuid: string;
+  aiButtonVisible: boolean;
+  currentStep: number;
+  setCurrentStep: (currentStep: number) => void;
+  enterpriseSlug: string;
+  targetSelector?: string;
+}
+
+const useAdminOnboardingTour = (
```

Comment:
Hooks are lifecycle function that utilize `useEffect` within, which isn't really the concern of these functions. What do you think about renaming them to normal functions (no `use-` prefix for functions or files) and instead of calling `useIntl()` just pass the `intl` object from the parent component?